chromium_src/components/bookmarks/browser/bookmark_model.cc

```diff
@@ -0,0 +1,25 @@
+
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../components/bookmarks/browser/bookmark_model.cc"
+
+namespace bookmarks {
+
+// Move bookmarks under "Other Bookmarks" permanent node to a same name folder
+// at the end of "Bookmark Bar" permanent node
+void BookmarkModel::MigrateOtherNode() {
```

Comment:
are any of these calls private? Can't we just pass the BookmarkModel in to a method somewhere else?

Reply:
addressed in 122272a4e2da3a516d56fe9672d56734a610cf59